src/components/shared/Input/TokenInput.vue

```diff
@@ -87,14 +87,15 @@
         </div>
 
         <div v-if="withSlider" class="input-line--footer-with-slider" @click="handleSliderFocus">
-          <div class="delimiter" />
```

Comment:
I think that it's better to just revert it without !important. What was the initial goal of that change?

Reply:
we have <s-divider /> component to split UI data, no need to create another custom one